src/lib/canonicalize-url.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Return a canonicalized version of the given URL.
+ *
+ * By default, the canonicalized URL should represent the same resource and
+ * typically de-reference to the same document (or a subpage of it).
+ *
+ * Canonicalization can be made a bit stronger through options, in particular
+ * to canonicalize dated W3C URLs to the Latest version, and to use a list of
+ * equivalent URLs (that the crawler typically generates).
+ */
+function canonicalizeUrl(url, options) {
+    options = options || {};
+
+    let canon = url.replace(/^http:/, 'https:')
```

Comment:
I know what @dontcallmedom would tell me if I was the one proposing these few lines of code that already exist in Reffy: "DRY!" ;)

That's fine with me, I just propose to add a comment so that we know that, should the code need to be updated, that update likely needs to be done in Reffy as well.

```suggestion
    // Same code as in Reffy:
    // https://github.com/w3c/reffy/blob/841db672190cf28658c29de3b8d7b8e28687ef47/src/postprocessing/annotate-links.js#L8-L19
    let canon = url.replace(/^http:/, 'https:')
```